docs/HazardAnalysis/HazardAnalysis.tex

```diff
@@ -38,36 +38,6 @@
 
 \pagenumbering{roman}
 
-\begin{table}[hp]
-\caption{Revision History} \label{TblRevisionHistory}
-\begin{tabularx}{\textwidth}{llX}
-\toprule
-\textbf{Date} & \textbf{Developer(s)} & \textbf{Change}\\
-\midrule
-16/10/23 & Ryan Yeh & Added List of Tables, List of Figures, and Failure Mode and Effect Analyis\\
-16/10/23 & Allan Fang & Added sections 1, 2, 3\\
-17/10/23 & Ryan Yeh & Added Critical Assumptions\\
-19/10/23 & Sawyer Tang & Initial Version of Safety and Security Requirements\\
-19/10/23 & Ryan Yeh & Added Roadmap\\
-01/04/24 & Sawyer Tang & Addressed Feedback: updated assumptions for camera availability and web service, updated HA2 to include\\
-\bottomrule
-\end{tabularx}
-\end{table}
-
-~\newpage
-
-\tableofcontents
-
-~\newpage
-
-\listoftables
-
-\listoffigures
-
-\newpage
-
-\pagenumbering{roman}
-
```

Comment:
Can you fix your deletion in the pdf PR? So for the duplicate revision table, you only want to delete lines `41 to 56` I believe and then delete lines 86 to 95 to get rid of the duplicate table of contents, list of figures and list of tables.

Essentially, the file's order should be table of contents, list of tables, list of figures and then the revision table, so verify that this is true after deletions.

Reply:
Oh I'm stupid, you could also just reorder the existing elements to match the order. Just make sure that everything compiles in the pdf, i.e. the table of contents and list of tables aren't blank.